public/css/restaurant-home.css

```diff
@@ -0,0 +1,150 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#F2F2F2;
+    --food-slider-bg:#171A29;
+    --text-col-white:#FFFFFF;
+    --text-col-gray:#9EA0A6;
+    --radio-color:#4f46e5;
+}
+.navbar{
+    padding: 1rem 1rem 0;
+}
+.nav-zindex{
+    z-index: 999;
+}
+
+.discount-container{
+    height: 35vh;
+    background: var(--food-slider-bg);
+    position: relative;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+}
+
+
+
+.hotel-img{
+    height: 15.625rem;
+    width: 20.625rem;
+    border-radius: 7px;
+    min-width: 14.375rem;
+    min-height: 14.375rem;
+    max-height: 14.375rem;
+    max-width: 21.875rem;
+    flex: 1;
+    object-fit: cover;
+}
+
+
+
+
+.hotelname{
```

Comment:
below for the hotel-list '-' is given but here '-' is not there, making the name a bit inconsistent

Reply:
will make the changes